### DISCUSSION TITLE
Periodic reports (email or slack)

### DISCUSSION THREAD

**Discussion by marcklingen** [langfuse/langfuse#1032] [Ideas]
Aggregated metrics for a high-level view of the Langfuse project

Ideas
- volume
- cost
- aggregated scores

This idea is super early and was raised by a couple of teams using Langfuse. Tracking this idea here, feel free to add your thoughts.

**Comment by Manouchehri** [comment]
It'd also be sick to get notifications of things like alerts on generations that triggered moderation / content management policies. =)

**Reply by emilyrohrbough** [reply]
+1 to alerts! Recently had an evaluator error for traces due to an API key issue, and it would have been great to have been notified there was an issue.

**Reply by marcklingen** [reply]
thanks for the +1 @emilyrohrbough

**Reply by marksalpeter** [reply]
Hi @Manouchehri and @emilyrohrbough! [Monitors](https://cloud.langfuse.com/project/~/monitors) just shipped. You can alert when moderation scores breach a threshold, or when an evaluator stops producing scores via a NO_DATA state. Alerts can be routed to Slack, webhook, or GitHub Actions ([docs](https://langfuse.com/docs/metrics/features/monitors)).

If you have any questions or feedback, please let us know.
